Userland/Libraries/LibDesktop/Screensaver.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022, Jelle Raaijmakers <jelle@gmta.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibDesktop/Screensaver.h>
+#include <LibGUI/Icon.h>
+
+namespace Desktop {
+
+constexpr int mouse_max_distance_move = 10;
```

Comment:
Not a huge deal, but file-locals like this should generally be `static` so they're only visible in this TU

Reply:
Both are made `static` now